apis/v1alpha1/ack-generate-metadata.yaml

```diff
@@ -1,14 +1,14 @@
 ack_generate_info:
-  build_date: "2021-06-21T19:55:19Z"
-  build_hash: d5797469ebb582083970136fc08f5de973e9ff1a
+  build_date: "2021-06-24T15:42:29Z"
+  build_hash: 9c35918f8a98b41ada10cb3aa460dbf463428463
   go_version: go1.16.4 linux/amd64
   version: v0.2.3
```

Comment:
shouldn't this be updated to 0.3.1? :) No need to fix this on this PR, just letting you know...

Reply:
This is the `ack-generate` Makefile :/ https://github.com/aws-controllers-k8s/code-generator/blob/main/Makefile#L9